### PR TITLE
Ship our local version of google-chrome and chromedriver

### PR DESCRIPTION
**Pull request by tuliocasagrande**
Fix functional test step `Then I should see the fancy interstitial`, that's not working properly with newer versions of chromedriver.

Since the debian repo doesn't provide old versions of chromium, we're [shipping][1] our own local copies:

```
$ chromedriver --version
ChromeDriver 2.27.440175 (9bc1d90b8bfa4dd181fbbf769a5eb5e575574320)
```
```
$ google-chrome --version
Google Chrome 54.0.2840.100
```

[1]: https://github.com/pixelated/pixelated-user-agent/releases/tag/1.0_beta1

### CODE REVIEW

**Comment by thaissiqueira** [comment]
Using the chromedriver branch I were able to run 'make tests_all' in a smoothly way. 
Sounds good to merge for me 👍

File: provisioning/modules/chromedriver/manifests/init.pp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  file { '/tmp/chromedriver_2.27_linux64.zip':
+    source => 'puppet:///modules/chromedriver/chromedriver_2.27_linux64.zip',
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+  }
+
+  exec { 'install_google_chrome':
```
Comment (tuliocasagrande) [inline provisioning/modules/chromedriver/manifests/init.pp]: Are you sure we can force a `/usr/bin/apt-get -y -f install` with this resource?

Comment (varac) [inline provisioning/modules/chromedriver/manifests/init.pp]: Oh, you're right, I oversaw that one. I don't know if this is possible, so we can stick to an exec here.
To don't run the exec every time pls add sth like this:

```
    unless => '/usr/bin/dpkg -l chromedriver > /dev/null 2>&1'
```

File: provisioning/modules/chromedriver/manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  exec { 'fetch_google_chrome':
+    command => "/usr/bin/wget https://github.com/pixelated/pixelated-user-agent/releases/download/${release}/${google_chrome}",
+    cwd     => '/var/tmp',
+    creates => "/var/tmp/${google_chrome}",
+  }
+
+  exec { 'unpack_chromedriver':
+    command => "/usr/bin/unzip ${chromedriver} -d /usr/local/bin/",
+    cwd     => '/var/tmp/',
+    creates => '/usr/local/bin/chromedriver',
```
Comment (varac) [inline provisioning/modules/chromedriver/manifests/init.pp]: This resource depends on Exec['fetch_chromedriver'] so pls add a `require` here.

File: provisioning/modules/chromedriver/manifests/init.pp [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  exec { 'install_google_chrome':
+    command => "/usr/bin/dpkg -i ${google_chrome} || /usr/bin/apt-get -y -f install",
+    cwd     => '/var/tmp/',
+    unless  => '/usr/bin/dpkg -l google-chrome-stable > /dev/null 2>&1',
+    require => [ Exec['apt_get_update'] ],
```
Comment (varac) [inline provisioning/modules/chromedriver/manifests/init.pp]: This resource depends on Exec['fetch_google_chrome'] so pls add it to the `require` array here.

**Pull request merged**